File: Division.c
#include <stdio.h>

int main(void){
    int x, y;

    printf("飴の個数を入力：");
    scanf("%d", &x);

    printf("人数を入力: ");
    scanf("%d", &y);

    int mul = x / y;
    int amari = x % y;

    printf("%d個の飴を%d人で分けると、一人あたり%d個で、%d個余ります\n", x,y,mul, amari);

    return 0;
}

File: Division.py
#!/usr/bin/python3
#SPDX-FileCopyrightText: 2023 Kaito Akiyama
#SPDX-License-Identifier: BSD-3-Clause

x = int(input("飴の個数を入力："))
y = int(input("人数を入力: "))

mul = x // y
amari = x % y

print(f"{x}個の飴を{y}人で分けると、一人あたり{mul}個で、{amari}個余ります")
